packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/About.1.doc.mdx

```diff
@@ -1,23 +1,105 @@
 
-## Getting Started
+<FixedWidthColumns />
+
+### Features
+Supports:
+* Arranging items in columns.
+* Fixed and responsive width columns.
+* `px`, `em`, and `rem` width units.
+* Vertical and horizontal flow.
+* Minimum and maximum column width for responsive columns set for all columns, or to each column individually.
+* Selectable cells.
+* Interactable elements in cells (one per cell).
+
+Refer to the [examples](/components/cerner-terra-framework-docs/compact-interactive-list/about#examples) section for detailed demonstration of the features supported by the Search Field.
+
+## Design Decisions
+
+The Compact Interactive List:
+  * Supports only one tab stop. When tabbing into the Compact Interactive List, focus will either go to the last visited cell, or the first cell upon initial entry.
+  While on the Compact Interactive List, `Tab` or `Shift+Tab` will exit the list.
+  * Does not maintain consumer data state. The state must be maintained by the consumer.
+  * Uses callbacks to communicate the actions taken by the user.
+  The callbacks are provided by the consumer and therefore the implementation is consumer specific.
+  It is up to the consumer to determine if and what actions should be taken.
+  For example, the user may click on a selectable cell in the Compact Interactive List which calls the _onCellSelect_ callback.
+  For business reasons, the consumer may decide not to honor that cell selection and instead does nothing resulting in the the cell not getting selected in the Worklist Data Grid.
+  * Supports single interactive element per cell.
+
+## Examples
+|Link to Example| Description|
+|-|-|
+|[FixedWidthColumns](/components/cerner-terra-framework-docs/compact-interactive-list/examples/fixed-width-columns)|An example of the Compact Interactive List with fixed width columns.|
+|[ResponsiveColumns](/components/cerner-terra-framework-docs/compact-interactive-list/examples/responsive-columns)|An example of the Compact Interactive List with responsive text columns.|
+|[ScalableUnits](/components/cerner-terra-framework-docs/compact-interactive-list/examples/scalable-units)|An example of the Compact Interactive List with width props set in units relative to the font size.|
+|[FlowDirection](/components/cerner-terra-framework-docs/compact-interactive-list/examples/flow-direction)|The examples of the Compact Interactive List with vertical and horizontal flow directions.|
+|[NumberOfColumns](/components/cerner-terra-framework-docs/compact-interactive-list/examples/number-of-columns)|An example of the Compact Interactive List placed inside of a parent element that uses width breakpoints to dynamically change the number of columns.|
+|[CellContent](/components/cerner-terra-framework-docs/compact-interactive-list/examples/cell-content)|Compact Interactive List cell content examples.|
+
+## Accessibility
+### Keyboard Interactions
+
+List of keyboard interactions supported by Compact Interactive List.
```

Comment:
Nit: wondering if a colon is more appropriate here:

```suggestion
List of keyboard interactions supported by Compact Interactive List:
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf